base/container_scripts/database_reset.sh

```diff
@@ -1,14 +1,7 @@
 #!/bin/bash
 
-# this gives us the reset_db django command
-pip show django-extensions || pip install django-extensions
-
 # the operation will error out unless all services are stopped
 ls /var/run/s6/services | egrep ^pulp | xargs -I {} s6-svc -d /var/run/s6/services/{}
 
 # wipe the database non-interactive
 yes yes | pulpcore-manager reset_db --user postgres
```

Comment:
I think django-admin (and as result pulpcore-manager) supports `--no-input` which could have been named `--force`.